datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemSliderViewHolderFactory.kt

```diff
@@ -74,3 +78,5 @@ internal object QuestionnaireItemSliderViewHolderFactory :
       }
     }
 }
+
+private const val SLIDER_DEFAULT_STEP_SIZE = 10
```

Comment:
let's make this 1. it's more usable by default.

```suggestion
private const val SLIDER_DEFAULT_STEP_SIZE = 1
```